pkg/provisioningv2/rke2/planner/planner.go

```diff
@@ -551,6 +548,9 @@ func (p *Planner) reconcile(controlPlane *rkev1.RKEControlPlane, tokensSecret pl
 		} else if !kubeletVersionUpToDate(controlPlane, entry.Machine) {
 			outOfSync = append(outOfSync, entry.Machine.Name)
 			messages[entry.Machine.Name] = append(messages[entry.Machine.Name], "waiting for kubelet to update")
+		} else if tierName == "control plane" && !controlPlane.Status.AgentConnected {
```

Comment:
is there supposed to be a space in control plane?

Reply:
Yes. I will update this to use `string` constants to be more understandable.